### PR TITLE
Update links to Sketch icon libraries

### PR DESCRIPTION
**Pull request by mariuszmickiewicz**

### CODE REVIEW

File: src/pages/guidelines/icons/usage.mdx [hunk 1]
```diff
@@ -45,15 +45,15 @@ important information.
 <Column colLg={4} colMd={4}  noGutterSm>
   <ResourceCard
     subTitle="Get the IBM Icons (16px, 20px) library"
```
Comment (janchild) [inline src/pages/guidelines/icons/usage.mdx:47]:
```suggestion
    subTitle="Get the IBM Icons (16px, 20px) library"
```
```suggestion
    subTitle="IBM Icons (16px, 20px) library"
```

File: src/pages/guidelines/icons/usage.mdx [hunk 2]
```diff
@@ -45,15 +45,15 @@ important information.
...
-    href="sketch://add-library/cloud/KW2yr"
+    href="sketch://add-library/cloud/028e0598-591e-428c-a490-f6ec64b15ea7"
     >
       <MdxIcon name="sketch" />
  </ResourceCard>
 </Column>
 <Column colLg={4} colMd={4}  noGutterSm>
   <ResourceCard
     subTitle="Get the IBM Icons (24px, 32px) library"
```
Comment (janchild) [inline src/pages/guidelines/icons/usage.mdx:55]:
```suggestion
    subTitle="IBM Icons (24px, 32px) library"
```

**Review by janchild — COMMENTED** [review]
@mariuszmickiewicz This looks good. I noticed that the wording was inconsistent here on these tiles (It should just be the noun listed, we don't need a verb.) so I took the opportunity to address. Thank you.

File: src/pages/guidelines/icons/usage.mdx (hunk 2 above)
Comment (mariuszmickiewicz) [reply src/pages/guidelines/icons/usage.mdx:55]: @janchild Thank you.
Could you look at this page too? https://www.carbondesignsystem.com/designing/kits/sketch
There are icons, but also theme libraries there and these you "Get".

Comment (janchild) [reply src/pages/guidelines/icons/usage.mdx:55]: @mariuszmickiewicz Thank you for pointing that out! We need to clean up that page too. I can open a PR for that. We have the verb in the introductory text above all of these, and so we don't need to repeat it on the tile. Even if we don't have introductory text though, including the verbs on the tiles just adds clutter.

**Review by laurenmrice — APPROVED** [review]
Thank you!

**Review by janchild — APPROVED** [review]
Thanks, @mariuszmickiewicz

**Review by tw15egan — APPROVED** [review]

**Pull request merged**